eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -244,7 +245,7 @@ class Setup(datadir: File,
         case (Block.RegtestGenesisBlock.hash, _) =>
           new FallbackFeeProvider(new ConstantFeeProvider(defaultFeerates) :: Nil, minFeeratePerByte)
         case (_, Bitcoind(bitcoinClient)) =>
-          new FallbackFeeProvider(new SmoothFeeProvider(new BitcoinCoreFeeProvider(bitcoinClient, defaultFeerates), smoothFeerateWindow) :: new SmoothFeeProvider(new BitgoFeeProvider(nodeParams.chainHash, readTimeout), smoothFeerateWindow) :: new SmoothFeeProvider(new EarnDotComFeeProvider(readTimeout), smoothFeerateWindow) :: Nil, minFeeratePerByte) // order matters!
+          new FallbackFeeProvider(new SmoothFeeProvider(new BitcoinCoreFeeProvider(bitcoinClient, defaultFeerates), smoothFeerateWindow) :: Nil, minFeeratePerByte)
```

Comment:
I think we can remove the FallbackFeeProvider in that case.

Reply:
In fact we can't, it's that `FallbackFeeProvider` that enforces the minimum feerate...
Once we've removed bitgo / earndotcom, we'll be able to clean this up a bit, but let's keep it for now?